ads/README.md

```diff
@@ -162,31 +162,25 @@ Once the request is processed the AMP runtime will try to accommodate this reque
 possible, but it will take into account where the reader is currently reading, whether the scrolling
 is ongoing and any other UX or performance factors.
 
-Ads can observe whether resize request were successful using the `window.context.onResizeSuccess` and `window.context.onResizeDenied` methods.
+The API will return a promise and the ads can observe whether resize request were successful by checking whether that promise resolves or rejects. `window.context.onResizeSuccess` and `window.context.onResizeDenied` methods are to be deprecated.
```

Comment:
Maybe link some sort of deprecation issue so that we can track when we started deprecation warning etc?